packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -386,3 +375,209 @@ test(`zcf.makeInvitation - customProperties overwritten`, async t => {
   });
   t.falsy(typeof details.handle === 'string');
 });
+
+test(`zcf.makeZCFMint - no keyword`, async t => {
+  const { zcf } = await setupZCFTest();
+  // @ts-ignore
+  await t.throwsAsync(() => zcf.makeZCFMint(), {
+    message: '(an undefined) must be a string\nSee console for error data.',
+  });
+});
+
+test(`zcf.makeZCFMint - keyword already in use`, async t => {
+  const { moolaIssuer } = setup();
+  const { zcf } = await setupZCFTest({ A: moolaIssuer });
+  await t.throwsAsync(() => zcf.makeZCFMint('A'), {
+    message:
+      'Keyword (a string) already registered\nSee console for error data.',
+  });
+});
+
+test(`zcf.makeZCFMint - bad keyword`, async t => {
+  const { zcf } = await setupZCFTest();
+  await t.throwsAsync(() => zcf.makeZCFMint('a'), {
+    message:
+      'keyword "a" must be ascii and must start with a capital letter.\nSee console for error data.',
+  });
+});
+
+test(`zcf.makeZCFMint - not a math kind`, async t => {
+  const { zcf } = await setupZCFTest();
+  // @ts-ignore
+  await t.throwsAsync(() => zcf.makeZCFMint('A', 'whatever'), {
+    message:
+      'unrecognized amountMathKind: (a string)\nSee console for error data.',
+  });
+});
+
+test(`zcf.makeZCFMint - NAT`, async t => {
+  const { zcf } = await setupZCFTest();
+  const zcfMint = await zcf.makeZCFMint('A', MathKind.NAT);
+  const issuerRecord = zcfMint.getIssuerRecord();
+  const expected = {
+    issuers: { A: issuerRecord.issuer },
+    brands: { A: issuerRecord.brand },
+    maths: { A: issuerRecord.amountMath },
+  };
+  await testTerms(t, zcf, expected);
+});
+
+test(`zcf.makeZCFMint - STRING_SET`, async t => {
+  const { zcf } = await setupZCFTest();
+  const zcfMint = await zcf.makeZCFMint('A', MathKind.STRING_SET);
+  const issuerRecord = zcfMint.getIssuerRecord();
+  const expected = {
+    issuers: { A: issuerRecord.issuer },
+    brands: { A: issuerRecord.brand },
+    maths: { A: issuerRecord.amountMath },
+  };
+  await testTerms(t, zcf, expected);
```

Comment:
It doesn't appear that this test ensures that there is anything in the result that is different from a NAT. Would you mind adding a check that the resulting amountMath is for STRING_SET?
(and for the SET math below.)

Reply:
Sure thing